### ISSUE TITLE
3.4.0: History stack copy/paste does not include lens correction module

### ISSUE THREAD

**Issue by gscholz** [darktable-org/darktable#7536]
**Describe the bug**

Here is a new bug I just found in 3.4.0 which was not existent in 3.2.1. The history stack copy/paste does not include the lens correction module.

**To Reproduce**

1. Edit image and activate "lens correction" module.
2. Copy history stack by activating "copy" button.
3. Paste history to an other image by activating "paste", which means paste _all_ module settings.
4. Result: all modules are copied and activated in second image but not the "lens correction" module.

Most probably the "paste" action does not work as intended as a selective paste by "paste parts..." allows to paste the lens correction settings after step 4. without further copy actions.

**Expected behavior**

Settings for the lens correction module should be pasted to new image and the module should be activated.

**Screenshots**

(none)

**Platform (please complete the following information):**
 - Darktable Version: 3.4.0
 - OS: Linux Mint 19.3
 - OpenCL: no
 - 

**Additional context**

Bug did not exist in version 3.2.1

**Comment by MStraeten** [comment]
see release notes (https://github.com/darktable-org/darktable/releases/tag/release-3.4.0)
```
The history copy has been changed to include only safe
modules. Modules like "raw white/black point" and "scale pixels" are
no longer copied, for example. To copy such modules, use the
"copy parts..." button. Note that the unsafe modules
are not selected by default in the resulting dialog.
```
use ctrl+shift+c to be able to select the non-safe modules

**Comment by johnny-bit** [comment]
Duplicate of #7504
